pkg/gateway/lb/usecase/vald.go

```diff
@@ -209,7 +240,7 @@ func (r *run) Start(ctx context.Context) (<-chan error, error) {
 	return ech, nil
 }
 
-func (r *run) PreStop(ctx context.Context) error {
+func (*run) PreStop(ctx context.Context) error {
```

Comment:
This change is for DeepSource warning.
https://deepsource.io/gh/vdaas/vald/run/9d9b8d27-8d57-4b75-b278-ec0b3983cf57/go/RVV-B0013